tracetools_launch/tracetools_launch/action.py

```diff
@@ -113,10 +113,11 @@ def __init__(
         self,
         *,
         session_name: SomeSubstitutionsType,
+        is_init_session: Union[bool, SomeSubstitutionsType] = False,
         append_timestamp: Union[bool, SomeSubstitutionsType] = False,
         base_path: Optional[SomeSubstitutionsType] = None,
         append_trace: Union[bool, SomeSubstitutionsType] = False,
-        events_ust: Iterable[SomeSubstitutionsType] = names.DEFAULT_EVENTS_ROS,
+        events_ust: Iterable[SomeSubstitutionsType] = None,
```

Comment:
I think this mechanism is good, but:

1. The documentation for `events_ust` should then mention that the default list of events used if `events_ust` is `None` (default) is `DEFAULT_EVENTS_ROS` if not `is_init_session` and otherwise it is `DEFAULT_INIT_EVENTS_ROS`.
1. The type annotation should be updated

```suggestion
        events_ust: Optional[Iterable[SomeSubstitutionsType]] = None,
```

Reply:
Fixed in https://github.com/ros2/ros2_tracing/pull/191/commits/05ce13ca04b4ec4a36626e504b9548de68bde8dc

Reply:
I think the documentation about the default values for `events_ust` that I mentioned above is still missing.

Reply:
Sorry, I missed it earlier.
Modified the doc string in https://github.com/ros2/ros2_tracing/pull/191/commits/cb868472c297c77198dcac004f3d1b42a17123f5